plugins/parquet/parquetembed.cpp

```diff
@@ -722,22 +722,15 @@ arrow::Status ParquetWriter::fieldToNode(const std::string &name, const RtlField
     case type_real:
         arrowFields.push_back(std::make_shared<arrow::Field>(name, arrow::float64()));
         break;
-    case type_string:
-        arrowFields.push_back(std::make_shared<arrow::Field>(name, arrow::utf8()));
-        break;
     case type_char:
-        arrowFields.push_back(std::make_shared<arrow::Field>(name, arrow::utf8()));
-        break;
+    case type_string:
     case type_varstring:
-        arrowFields.push_back(std::make_shared<arrow::Field>(name, arrow::utf8()));
-        break;
     case type_qstring:
         arrowFields.push_back(std::make_shared<arrow::Field>(name, arrow::utf8()));
         break;
-    case type_unicode:
-        arrowFields.push_back(std::make_shared<arrow::Field>(name, arrow::utf8()));
-        break;
     case type_utf8:
+    case type_unicode:
+    case type_varunicode:
```

Comment:
This is more stylistic than anything else, but I would combine all of the case options that result in exactly the same result (arrow::utf8).  At least for all these string-like data types.

Reply:
Fixed.